tdrs-frontend/src/actions/reports.js

```diff
@@ -122,6 +129,69 @@ export const upload = ({ file, section }) => async (dispatch) => {
   return true
 }
 
+export const submit = ({
+  formattedSections,
+  logger,
+  quarter,
+  setLocalAlertState,
+  stt,
+  uploadedFiles,
+  user,
+  year,
+}) => async (dispatch) => {
+  const submissionRequests = uploadedFiles.map((file) => {
```

Comment:
Note to self: refactor this to call a reducer to set the ID of the uploaded file in the state - this will allow the download button to properly show up after submission without having to refresh and allows us to more accurately track the state of files (if id != null then we know it has been submitted).

If not in this PR, a follow on issue should be created.

Reply:
Follow on issue: https://github.com/raft-tech/TANF-app/issues/1108